include/smack/SmackRep.h

```diff
@@ -83,7 +89,16 @@ class SmackRep {
   Program& getProgram() { return program; }
 
 private:
-  void addInit(unsigned region, const Expr* addr, const llvm::Constant* val);
+  // 
+  // Pointer size: 32
+  //
+  static const unsigned ptrsize;
+  //
+  // A flag for bit-vector
+  //
+  static bool BIT_VECTOR;
```

Comment:
SmackOptions already provides (static) variables for these quantities. We could also switch those on in the code by a “possibly complex Boolean expression of multiple factors” (though I don't see why we could need a complex expression…). Is there any real reason not to just reuse those existing SmackOptions variables?